src/app/ngrx/reducers/episode-metrics.reducer.ts

```diff
@@ -71,35 +71,29 @@ export function EpisodeMetricsReducer(state: EpisodeMetricsModel[] = initialStat
         return newState;
       }
       break;
-    case ACTIONS.ActionTypes.CASTLE_EPISODE_ALL_TIME_METRICS_SUCCESS:
-      if (action instanceof ACTIONS.CastleEpisodeAllTimeMetricsSuccessAction) {
-        const { id, seriesId } = action.payload.episode;
-        const epIdx = episodeIndex(state, id, seriesId);
+    // TODO: #141 & #142 should remove from this state and combine with performance metrics state
```

Comment:
When I'm able to create the selectors for the chart (and table) data and clean up those components, we'll get the total all time number from the performance metrics and drop it from this reducer.